week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>I might recommend centering the equations image, you can do this by typing the following as markdown (in its own cell):</p><pre class="ql-syntax" spellcheck="false">&lt;center&gt;&lt;img src="Pythagorean(1).gif" width=600 height =600/&gt;&lt;center/&gt;
</pre><p> </p><p>And then consider editing the text and equations like how Greg demonstrated in class today. It just makes the cell a bit cleaner and easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>